server/handlers/design_engine_handler.go

```diff
@@ -146,7 +146,14 @@ func (h *Handler) PatternFileHandler(
 		"summary": response,
 	}
 
-	event := eventBuilder.WithSeverity(events.Informational).WithDescription(description).WithMetadata(metadata).Build()
+	serverURL, _ := r.Context().Value(models.MesheryServerURL).(string)
+	
+	if action == "deploy" {
+		viewLink := fmt.Sprintf("%s/extension/meshmap?mode=visualize&design=%s", serverURL, patternID)
+		description = fmt.Sprintf("%s. View deployed design %s", description, viewLink)
+	}
+
+	event := eventBuilder.WithSeverity(events.Success).WithDescription(description).WithMetadata(metadata).Build()
```

Comment:
Should this be an Info severity? @MUzairS15

Reply:
This event is emitted when the design gets deployed and the metadata of the event contains which comps were deployed and their type, so it should be of severity success (currently we have `info`) but `success` makes more sense.

Reply:
Oh, ok. 👍